testsuite/gnat2goto/tests/nat_and_pos/nat_and_pos.adb

```diff
@@ -8,16 +8,14 @@ procedure Nat_And_Pos is
      with Annotate => (ASVAT, In_Type_Function),
      Import;
 begin
-   Var := -1;
-   pragma Assert (In_Type_Natural (Var));
```

Comment:
Why remove these assertions?

Reply:
The test has been reordered so that only the last test fails. the test of In_Type_Natural (-1) has been moved to the end of the tests.

Reply:
Ah, yes, if they are checks on constants that makes sense.